internal/msgs/en_config_descriptions.go

```diff
@@ -27,17 +27,19 @@ var ffc = func(key, translation string, fieldType string) i18n.ConfigMessageKey
+	ConfigEthereumGasEstimationFactor              = ffc("config.connector.gasEstimationFactor", "The factor to apply to the gas estimation to determine the gas limit", "float")
+	ConfigBlockCacheSize                           = ffc("config.connector.blockCacheSize", "Maximum of blocks to hold in the block info cache", i18n.IntType)
+	ConfigBlockPollingInterval                     = ffc("config.connector.blockPollingInterval", "Interval for polling to check for new blocks", i18n.TimeDurationType)
+	ConfigEventsBlockTimestamps                    = ffc("config.connector.events.blockTimestamps", "Whether to include the block timestamps in the event information", i18n.BooleanType)
+	ConfigEventsCatchupPageSize                    = ffc("config.connector.events.catchupPageSize", "Number of blocks to query per poll when catching up to the head of the blockchain", i18n.IntType)
+	ConfigEventsCatchupThreshold                   = ffc("config.connector.events.catchupThreshold", "How many blocks behind the chain head an event stream or listener must be on startup, to enter catchup mode", i18n.IntType)
+	ConfigEventsCatchupDownscaleRegex              = ffc("config.connector.events.catchupDownscaleRegex", "An error pattern to check for from JSON/RPC providers if they limit response sizes to eth_getLogs(). If an error is returned from eth_getLogs() and that error matches the configured pattern, the number of logs requested (catchupPageSize) will be reduced automatically.", "string")
+	ConfigEventsCheckpointBlockGap                 = ffc("config.connector.events.checkpointBlockGap", "The number of blocks at the head of the chain that should be considered unstable (could be dropped from the canonical chain after a re-org). Unless events with a full set of confirmations are detected, the restart checkpoint will this many blocks behind the chain head.", i18n.IntType)
+	ConfigEventsFilterPollingInterval              = ffc("config.connector.events.filterPollingInterval", "The interval between polling calls to a filter, when checking for newly arrived events", i18n.TimeDurationType)
+	ConfigTxCacheSize                              = ffc("config.connector.txCacheSize", "Maximum of transactions to hold in the transaction info cache", i18n.IntType)
+	ConfigMaxConcurrentRequests                    = ffc("config.connector.maxConcurrentRequests", "Maximum of concurrent requests to be submitted to the blockchain", i18n.IntType)
+	ConfigAllowNonStandardBlockHashSize            = ffc("config.connector.allowNonStandardBlockHashSize", "Whether to permit block hases of a non-standard (i.e. not 32 bytes) length", i18n.BooleanType)
+	ConfigNonStandardBlockHashSizeResolutionMethod = ffc("config.connector.nonStandardBlockHashSizeResolutionMethod", "How to resolve block hashes that are non-standard (i.e. not 32 bytes) in length. Currently only 'truncate' is supported.", i18n.StringType)
```

Comment:
I'm not sure I understand yet the intention here, and that suggests to me users of this setting might struggle to.

Is the intention here `blockHashByteLength`?

e.g. you are instructing the whole code stack that all block hashes should be X bytes, where the default is 32?

If so, what code is affected that needs this? And why? 

Will try and answer from reading through the rest of the code